api/pkg/apis/v1alpha1/managers/activations/activations-manager.go

```diff
@@ -208,6 +223,11 @@ func (t *ActivationsManager) ReportStatus(ctx context.Context, name string, name
 
 	current.UpdateTime = time.Now().Format(time.RFC3339) // TODO: is this correct? Shouldn't it be reported?
 	activationState.Status = &current
+	if activationState.ObjectMeta.Labels == nil {
+		activationState.ObjectMeta.Labels = make(map[string]string)
+	}
+	// label doesn't allow space, so remove space
+	activationState.ObjectMeta.Labels["statusMessage"] = strings.ReplaceAll(current.Status.String(), " ", "")
```

Comment:
why put statusmessage in labels?

Reply:
1. Now we call k8s server for list call which only support filtering on limited fields (field filter is only supported in controller runtime cache)
2. We need to use statusMessage to get activations in running state